src/samples/article-poa/v2/complete.json

```diff
@@ -120,6 +120,7 @@
         }
     ],
     "pdf": "https://elifesciences.org/content/5/e14107.pdf",
```

Comment:
can these samples be modernized with journal-style URLs?

Reply:
They should be buckets if they're to be 'real'.

Reply:
Could make them clearly fake rather than fake but redirecting to the wrong page? (https://elifesciences.org/content/5/e14107.pdf goes to https://elifesciences.org/articles/14107)

Reply:
In Crossref deposits, the XML and PDF URIs are set using the format in a configuration file (https://github.com/elifesciences/elife-crossref-xml-generation/blob/develop/crossref.cfg#L36-L37) loading the version's files from the CDN. If that is useful in the sample data at all.

I guess the value can be populated by ``bot-lax-adaptor`` when the property is available in the schema.

Reply:
It should be, it's a predictable URL akin to the PDF

Reply:
Will leave the sample change for a separate PR.